### DISCUSSION TITLE
Prompted to Enter Password for Jupyter Server on VSCode Launch

### DISCUSSION THREAD

**Discussion by snowyday** [microsoft/vscode-jupyter#15538] [Questions and Answers]
I'm encountering an issue with VSCode where I am prompted to enter the password for the Jupyter Server every time I launch the application, even when I do not open any notebooks.

Here are the details of my setup:

Operating System: macOS Sonoma
VSCode Version: 1.88.0

I am using Jupyter on a remote machine, to which VSCode connects for my workflow. Regardless of whether or not I interact with any Jupyter notebooks, the password prompt for the remote server's Jupyter Server appears with each launch. 

This behavior is consistent and reproducible on my end. I am looking for a way to either bypass this password prompt upon launch or a method to fix this so that the prompt only appears when I attempt to access a Jupyter notebook.

Any assistance on this matter would be greatly appreciated.

**Comment by markschad** [comment]
Can also reproduce this, very annoying

**Comment by diego898** [comment]
same for me on ubuntu - is there an issue for this?

**Comment by kkew3** [comment]
Same with VScode 1.90.2 (Universal) on macOS Monterey. I also notice that the prompt pops out even after reloading any extensions.

**Comment by Habush** [comment]
Same here as on Windows 11, VSCode 1.90.2.

**Comment by jstnjns** [comment]
also experiencing this on mac os 14.5, vscode 1.92.1

**Comment by diego898** [comment]
Does anyone know if there's an actual issue to fix this behavior?

**Comment by risan-raja** [comment]
Even I am experiencing this issue on Mac

**Comment by xixiha5230** [comment]
Fixed after update to VSCode 1.94.2. on Windows 10,

**Reply by Zhangweich** [reply]
Still unfixed in1.95.2 on windows 10, very annoying...

**Comment by thinkORo** [comment]
I have had the same problem everytime I opened VSCode if, in addition, a py-file was opened as well. The easiest thing to do was to delete the list of known Jupyter servers from the “cache”.

Simply press Ctrl+Shift+P or, on the Mac, Cmd+Shift+P and select “Jupyter: Clear Jupyter Remote Server List”.

At least for me, the message was then gone.

**Reply by seetimee** [reply]
maybe it's because the remote kernels need to be authorized to make them worked at first. It's fixed after I deleted my cached remote Jupyter servers
